src/extension/ncdfgenerator/src/main/java/au/org/emii/ncdfgenerator/Main.java

```diff
@@ -88,7 +92,20 @@ public static void main(String[] args) {
             System.exit(1);
         }
 
+        JDBCDataStore dataStore = null;
+
         try {
+            Map<String, Object> dsParams = new HashMap<String, Object>();
+            dsParams.put("dbtype", "postgis");
+            dsParams.put("host", "dbprod.emii.org.au");
+            dsParams.put("port", 5432);
+            dsParams.put("database", "harvest?loginTimeout=1000&ssl=true&sslfactory=org.postgresql.ssl.NonValidatingFactory");
```

Comment:
Should we have these values here hard coded - the existing code for connection takes command line parameters